docs/content/start/telemetry.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Quick Start
+menuPosition: 4
+codeCopyButton: true
+aliases:
+  - "/docs/get-started/telemetry/"
+---
```

Comment:
Not really sure if I need to include this, but given that other files in this folder has it, I added it for consistency.